codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/HttpBoundProtocolPayloadGenerator.kt

```diff
@@ -156,39 +167,67 @@ class HttpBoundProtocolPayloadGenerator(
     private fun RustWriter.serializeViaEventStream(
         operationShape: OperationShape,
         memberShape: MemberShape,
-        serializerGenerator: StructuredDataSerializerGenerator
+        serializerGenerator: StructuredDataSerializerGenerator,
+        outerName: String,
     ) {
         val memberName = symbolProvider.toMemberName(memberShape)
         val unionShape = model.expectShape(memberShape.target, UnionShape::class.java)
 
+        val contentType = when (target) {
+            CodegenTarget.CLIENT -> httpBindingResolver.requestContentType(operationShape)
+            CodegenTarget.SERVER -> httpBindingResolver.responseContentType(operationShape)
+        }
```

Comment:
I don't find it in the spec, is it a custom header used by some services? If so, I should remove it rather than adjusting it. On some services I see it's `application/vnd.amazon.event-stream` (notice the `-`)

Reply:
Discussed offline and created https://github.com/awslabs/smithy/issues/1287 to document them. Fixed in this PR